web/themes/custom/usagov/scripts/telephoneForMobilePage.js

```diff
@@ -0,0 +1,15 @@
+(function ($) {
```

Comment:
It looks like there are two versions of this script, and this one is not used -- if so, let's delete it.

Reply:
Will do!